src/seraphis_main/scan_balance_recovery_utils.cpp

```diff
@@ -378,10 +379,15 @@ bool try_find_legacy_enotes_in_tx(const rct::key &legacy_base_spend_pubkey,
         legacy_main_enote_ephemeral_pubkey,
         legacy_additional_enote_ephemeral_pubkeys);
 
-    // 2. check if there are a valid number of additional enote ephemeral pubkeys
+    // 2. check if there are a valid number of components
+    // a. additional enote ephemeral pubkeys
     if (legacy_additional_enote_ephemeral_pubkeys.size() > 0 &&
         legacy_additional_enote_ephemeral_pubkeys.size() != enotes_in_tx.size())
         return false;
+    
```

Comment:
Nitpick: trailing white space (for me it doesn't show on GitHub, but in local editor).